Packs/FeedAzureADConnectHealth/pack_metadata.json

```
@@ -0,0 +1,20 @@
+{
+    "name": "Microsoft Azure AD Connect Health Feed",
+    "description": "Indicator feed from Microsoft Azure AD Connect Health endpoints",
```

can we add some more data indicating that these IOCs are good?
```suggestion
    "description": "Indicator feed from Microsoft Azure AD Connect Health endpoints, fetching URLs and DomainGlobs used by Azure AD, with which you can create a list (allowlist, EDL, etc.) for your SIEM or firewall service to ingest and apply to its policy rules.",
```

---

Done